flowermd/base/system.py

```diff
@@ -668,10 +671,11 @@ def __init__(
         self.packing_expand_factor = packing_expand_factor
         self.edge = edge
         self.overlap = overlap
+        self.overlap = seed
         self.fix_orientation = fix_orientation
         super(Pack, self).__init__(molecules=molecules, base_units=base_units)
 
-    def _build_system(self):
+    def _build_system(self, **kwargs):
```

Comment:
We'll have to put this in the `__init__()` method for `Pack` after all of the other parameters. Assign it to a class variable with `self` then use that in `fill_box`